src/ds/d500/d500-options.h

```diff
@@ -76,6 +76,24 @@ namespace librealsense
         const char* _description;
     };
 
+    class temperature_xu_option : public uvc_xu_option<int16_t>, 
+        public readonly_option
+    {
+    public:
+
+        explicit temperature_xu_option(const std::weak_ptr< uvc_sensor >& ep,
+            platform::extension_unit xu,
+            uint8_t id,
+            std::string description,
+            bool allow_set_while_streaming = true);
```

Comment:
Not a big deal, but this is not needed, can be set to false directly in the c'tor